zebra-chain/src/transaction.rs

```diff
@@ -479,6 +479,29 @@ impl Transaction {
 
     // sprout
 
+    /// Returns the Sprout `JoinSplit`s in this transaction, regardless of version.
+    pub fn sprout_joinsplits(
```

Comment:
Naming: `joinsplit_count` and `sprout_nullifiers` return both BCTV14 and Groth16 JoinSplits.

Can we rename this method to make it clear it only returns Groth16 JoinSplits?

```suggestion
    pub fn sprout_groth16_joinsplits(
```

Reply:
I think this is done in c82bc265dde535fb93d2ee1529513e0d09f84b3c